packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/translations/Messages_fr_FR.json

```diff
@@ -12,6 +12,7 @@
   "telephony_line_phone_actions_line_order_accessories": "Commander des accessoires",
   "telephony_line_phone_actions_line_phonebook": "Carnets de contacts",
   "telephony_group_line_phone_section_back_link": "Retour à la gestion du téléphone",
```

Comment:
This translation key could be removed no ?

Reply:
i prefere keep it because it have other usage i think and i do not want to break something in fix PR :)